app/components/cases/edit-case-with-confidential.hbs

```diff
@@ -0,0 +1,59 @@
+<Auk::Modal>
+  <Auk::Modal::Header
+          @title={{t "edit-case"}}
+          @onClose={{@onClose}}
+  />
+  <Auk::Modal::Body>
+    {{#if hasError}}
+      <Auk::Alert
+        @skin="error"
+        @icon="alert-triangle"
+        @title={{t "modal-error-alert-title"}}
+        @message={{t "modal-error-alert-message"}}
+      />
+    {{/if}}
+    <div class="auk-u-mb-4">
+      <div class="auk-form-group">
+        <Auk::Label>{{t "confidential"}}</Auk::Label>
+        <Auk::CheckboxList>
+          <Auk::Checkbox @name="isConfidential" @checked={{this.isConfidential}}
+            {{on "input" this.toggleConfidential}} />
+        </Auk::CheckboxList>
+      </div>
+    </div>
+    <div class="auk-form-group {{getClassForShortTitle}}">
+      <Auk::Label>{{t "short-title-case"}}</Auk::Label>
+      <Auk::Textarea @rows="2" @value={{this.shortTitle}} />
+    </div>
+  </Auk::Modal::Body>
+  <Auk::Modal::Footer @custom={{true}}>
+    <Auk::Toolbar>
+      <Auk::Toolbar::Group @position="left">
+        <Auk::Toolbar::Item>
+          <Auk::Button
+                  @skin="borderless"
+                  disabled={{this.save.isRunning}}
+            {{on "click" @onClose}}
+          >
+            {{t "cancel"}}
+          </Auk::Button>
+        </Auk::Toolbar::Item>
```

Comment:
Er is een kortere variant van de `Auk::Modal::Footer` zodat je de annuleren-button aan de linkerkant niet zelf moet specifieren maar gewoon een `@onCancel` en `@cancelDisabled` argument kan meegeven.